src/aws/target.ts

```diff
@@ -32,9 +50,9 @@ export default class Target {
           ScalableDimension: 'lambda:function:ProvisionedConcurrency',
           ServiceNamespace: 'lambda',
           RoleARN: {
-            'Fn::Sub':
-              'arn:aws:iam::${AWS::AccountId}:role/aws-service-role/lambda.application-autoscaling.amazonaws.com/AWSServiceRoleForApplicationAutoScaling_LambdaConcurrency',
+            'Fn::Sub': 'arn:aws:iam::${AWS::AccountId}:role/aws-service-role/lambda.application-autoscaling.amazonaws.com/AWSServiceRoleForApplicationAutoScaling_LambdaConcurrency',
           },
+          ScheduledActions
```

Comment:
Instead of creating this object, can we use a ternary, like in the custom metric defaults, in my other pr?

```typescript
ScheduledActions: this.data.scheduledActions
  ? this.getSchedulesActions()
  : undefined,
```

Reply:
yeap, np
done